netlify/edge-functions/placeholder-fallback.ts

```diff
@@ -0,0 +1,38 @@
+import type { Context } from "https://edge.netlify.com";
```

Comment:
We can't use edge functions. These will costs us a lot of money.

Reply:
Ugh... I just stupidly assumed the open source plan was unlimited invocations.  Apparently not even the enterprise plan is unlimited.

We could potentially keep the invocations under the  limit by caching the function responses (invocation only occurs when it runs per their docs).  Search parameter would then be out though as they only cache based on path name.  Theoretically, invocations would then = number of files x 2 x deploys per month.

Working within the config file constraints, the best alternative would be to switch the redirects to rewrites and lose the ability to vary the cache control header.

Reply:
> This is a frontend problem, solve it in the frontend 

The frontend cannot control responses from https://brands.home-assistant.io.  A solution there means unnecessary extra code, repeating the functionality here, and, most notably, wasted network requests that reduce performance.

> (that said, I don't see a problem at all, so the "problem" in that sentence is overkill).

I did my best to explain it in the issue.  "Sorry to say", but you not seeing a problem doesn't mean it doesn't exist.  If you'd like to see it, pull the frontend PR, test it, and look at the errors in the browser console.